website/docs/addon_motionbuilder_admin.md

```diff
@@ -10,6 +10,27 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 {versions.MotionBuilder_Badge}
 </ReactMarkdown>
 
-:::note Work in progress
-This part of documentation is still work in progress.
+:::caution Unknown support for Motion Builder 2023 and below
+
+The AYON Motion Builder addon was initially developed with Motion Builder 2024 and 2025.
+Hence functionality was not tested in older versions than 2024 and may not work.
+:::
+
+## Supported Workflows
+
+The MotionBuilder addon currently enables you to import models, rigs, animations, and cameras. It also supports exporting animations and managing workfiles through saving, opening, and publishing.
+
+:::info Motion Builder Addon Settings
+The Motion Builder addon does not include individual settings, so it won't be listed in your studio's addon settings. However, this does not affect the addon's ability to utilize the core settings.
```

Comment:
```suggestion
The MotionBuilder addon does not include individual settings, so it won't be listed in your studio's addon settings. However, this does not affect the addon's ability to utilize the core settings. To detect whether `ayon-motionbuilder` is in your bundle check the Bundles page (shortcut B+B).
```

Reply:
Thank you!
https://github.com/ynput/ayon-documentation/pull/221/commits/ccd92ca1db1d001f4a9d24c480a2ad0f0a335de0